### PR TITLE
Added additional NIC devices to qemu backend

### PR DESCRIPTION
**Pull request by dzyuzin**
Added new variables NICTYPE_1, NICTYPE_2 (only TAP devices), TAPDEV_1, TAPDEV_2, NICMAC_1 and NICMAC_2

### CODE REVIEW

File: doc/backend_vars.asciidoc [hunk 1]
```diff
@@ -50,8 +50,12 @@ MAKETESTSNAPSHOTS;integer;0;save snapshot for each test in qcow image
 MULTINET;boolean;0;VM will have two virtual NIC. Can't be used with NICTYPE tap
 MULTIPATH;boolean;0;Add HDD drives as multipath devices. Override HDDMODEL to virtio-scsi-pci
 NICMAC;any MAC address;52:54:00:12:34:56;MAC address to be assigned to virtual network card
+NICMAC_1;any MAC address;52:54:01:12:34:56;MAC address to be assigned to virtual network card
+NICMAC_2;any MAC address;52:54:02:12:34:56;MAC address to be assigned to virtual network card
```
Comment (coolo) [inline doc/backend_vars.asciidoc]: it's the same default MAC address?

Comment (coolo) [inline doc/backend_vars.asciidoc]: ah, tiny difference - I spotted it :)

**Comment by coolo** [comment]
run ./tools/tidy before commit

**Comment by aaannz** [comment]
I must say I don't like this approach very much. I would like to see something more general like iterating through NICTYPE_%n etc. And also why limit them to TAP. We can eventually remove MULTINET (or simulate by adding multiple NICTYPEs for backwards compatibility).
I don't have particular use case in mind, just doing this seems strange to me.

Also see `git rebase -p` from upstream might help to remove the merge commit from PR.

**Comment by nadvornik** [comment]
Here is an alternative implementation:
https://github.com/os-autoinst/os-autoinst/pull/319

**Pull request closed without merging**